__tests__/email.js

```diff
@@ -0,0 +1,13 @@
+import { reformatEmail } from "../utils/email.js";
+
+describe("", () => {
```

Comment:
The "describe" block allows us to give a name to a group of tests. We should specify a name here, or, if we don't need to group the tests together, we can hoist the tests to the top level.

```suggestion
describe("emails", () => {
```